test/base-config-fonturl.js

```diff
@@ -10,12 +11,12 @@ tape('basic configuration: check fontURL', function (t) {
         format: "TeX",
         css: true
     }, function (result, data) {
-        t.ok(result.css.indexOf('https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/fonts/HTML-CSS') > -1, 'Default fontURL');
+        t.ok(result.css.indexOf('https://cdnjs.cloudflare.com/ajax/libs/mathjax/' + mjVersion + '/fonts/HTML-CSS') > -1, 'Default fontURL');
```

Comment:
The other option, I suppose, is to do a regex search with `mjVersion` replaced by something like `\d+\.\d+\.\d+`.  But I'm OK with the lookup as is.

Reply:
Ah, nice idea. I somewhat favor this one but I'm ok either way.

Reply:
No problem.  Keep it as is, then.